### PR TITLE
Assigned services, characteristics and descriptors updated 

### PR DESCRIPTION
**Pull request by zakorgy**
in bluetoothuuid.rs

### CODE REVIEW

File: components/script/dom/bluetoothuuid.rs [hunk 1]
```diff
@@ -19,26 +19,245 @@ pub struct BluetoothUUID {
...
+    ("org.bluetooth.characteristic.weight_scale_feature", 0x2a9e_u32),
+    ("org.bluetooth.characteristic.wind_chill", 0x2a79_u32)
 ];
 
 const BLUETOOTH_ASSIGNED_DESCRIPTORS: &'static [(&'static str, u32)] = &[
-//TODO(zakorgy) create all the descriptors
 //https://developer.bluetooth.org/gatt/services/Pages/ServicesHome.aspx
-    ("org.bluetooth.descriptor.gatt.characteristic_extended_properties", 0x2900_u32),
-    ("org.bluetooth.descriptor.gatt.characteristic_user_description", 0x2901_u32)
+    ("org.bluetooth.descriptorgatt.characteristic_extended_properties", 0x2900),
```
Comment (dati91) [inline components/script/dom/bluetoothuuid.rs]: If the others are `*_u32`, shouldn't we use the `u32` here as well?

File: components/script/dom/bluetoothuuid.rs [hunk 2]
```diff
@@ -19,26 +19,245 @@ pub struct BluetoothUUID {
...
+    ("org.bluetooth.service.object_transfer", 0x1825_u32),
+    ("org.bluetooth.service.phone_alert_status", 0x180e_u32),
+    ("org.bluetooth.service.pulse_oximeter", 0x1822_u32),
+    ("org.bluetooth.service.reference_time_update", 0x1806_u32),
+    ("org.bluetooth.service.running_speed_and_cadence", 0x1814_u32),
+    ("org.bluetooth.service.scan_parameters", 0x1813_u32),
+    ("org.bluetooth.service.transport_discovery", 0x1824),
+    ("org.bluetooth.service.tx_power", 0x1804_u32),
+    ("org.bluetooth.service.user_data", 0x181c_u32),
+    ("org.bluetooth.service.weight_scale", 0x181d_u32)
```
Comment (mmatyas) [inline components/script/dom/bluetoothuuid.rs]: Protip: you can put `,` to the end of every line

File: components/script/dom/bluetoothuuid.rs [hunk 3]
```diff
@@ -19,26 +19,245 @@ pub struct BluetoothUUID {
...
 ];
 
 const BLUETOOTH_ASSIGNED_CHARCTERISTICS: &'static [(&'static str, u32)] = &[
-//TODO(zakorgy) create all the characteristics
 //https://developer.bluetooth.org/gatt/services/Pages/ServicesHome.aspx
     ("org.bluetooth.characteristic.aerobic_heart_rate_lower_limit", 0x2a7e_u32),
     ("org.bluetooth.characteristic.aerobic_heart_rate_upper_limit", 0x2a84_u32),
-    ("org.bluetooth.characteristic.battery_level", 0x2a19_u32)
```
Comment (mmatyas) [inline components/script/dom/bluetoothuuid.rs]: Same

File: components/script/dom/bluetoothuuid.rs [hunk 4]
```diff
@@ -19,26 +19,245 @@ pub struct BluetoothUUID {
 }
 
 const BLUETOOTH_ASSIGNED_SERVICES: &'static [(&'static str, u32)] = &[
-//TODO(zakorgy) create all the services
 //https://developer.bluetooth.org/gatt/services/Pages/ServicesHome.aspx
```
Comment (mmatyas) [inline components/script/dom/bluetoothuuid.rs:21]: Shouldn't this be above `const BLUETOOTH_ASSIGNED_SERVICES:` ?

File: components/script/dom/bluetoothuuid.rs [hunk 5]
```diff
@@ -19,26 +19,245 @@ pub struct BluetoothUUID {
...
+    ("org.bluetooth.service.scan_parameters", 0x1813_u32),
+    ("org.bluetooth.service.transport_discovery", 0x1824),
+    ("org.bluetooth.service.tx_power", 0x1804_u32),
+    ("org.bluetooth.service.user_data", 0x181c_u32),
+    ("org.bluetooth.service.weight_scale", 0x181d_u32)
 ];
 
 const BLUETOOTH_ASSIGNED_CHARCTERISTICS: &'static [(&'static str, u32)] = &[
-//TODO(zakorgy) create all the characteristics
 //https://developer.bluetooth.org/gatt/services/Pages/ServicesHome.aspx
```
Comment (mmatyas) [inline components/script/dom/bluetoothuuid.rs:60]: Same

File: components/script/dom/bluetoothuuid.rs [hunk 6]
```diff
@@ -19,26 +19,245 @@ pub struct BluetoothUUID {
...
+    ("org.bluetooth.characteristic.waist_circumference", 0x2a97_u32),
+    ("org.bluetooth.characteristic.weight", 0x2a98_u32),
+    ("org.bluetooth.characteristic.weight_measurement", 0x2a9d_u32),
+    ("org.bluetooth.characteristic.weight_scale_feature", 0x2a9e_u32),
+    ("org.bluetooth.characteristic.wind_chill", 0x2a79_u32)
 ];
 
 const BLUETOOTH_ASSIGNED_DESCRIPTORS: &'static [(&'static str, u32)] = &[
-//TODO(zakorgy) create all the descriptors
 //https://developer.bluetooth.org/gatt/services/Pages/ServicesHome.aspx
```
Comment (mmatyas) [inline components/script/dom/bluetoothuuid.rs]: Same

File: components/script/dom/bluetoothuuid.rs [hunk 7]
```diff
@@ -19,26 +19,245 @@ pub struct BluetoothUUID {
...
-    ("org.bluetooth.descriptor.gatt.characteristic_extended_properties", 0x2900_u32),
-    ("org.bluetooth.descriptor.gatt.characteristic_user_description", 0x2901_u32)
```
Comment (mmatyas) [inline components/script/dom/bluetoothuuid.rs]: Same

File: components/script/dom/bluetoothuuid.rs [hunk 8]
```diff
@@ -19,26 +19,245 @@ pub struct BluetoothUUID {
...
+    ("org.bluetooth.descriptorgatt.characteristic_aggregate_format", 0x2905),
+    ("org.bluetooth.descriptorvalid_range", 0x2906),
+    ("org.bluetooth.descriptorexternal_report_reference", 0x2907),
+    ("org.bluetooth.descriptorreport_reference", 0x2908),
+    ("org.bluetooth.descriptornumber_of_digitals", 0x2909),
+    ("org.bluetooth.descriptorvalue_trigger_setting", 0x290a),
+    ("org.bluetooth.descriptores_configuration", 0x290b),
+    ("org.bluetooth.descriptores_measurement", 0x290c),
+    ("org.bluetooth.descriptores_trigger_setting", 0x290d),
+    ("org.bluetooth.descriptortime_trigger_setting", 0x290e)
```
Comment (mmatyas) [inline components/script/dom/bluetoothuuid.rs]: Same

File: components/script/dom/bluetoothuuid.rs (hunk 6 above)
Comment (mmatyas) [inline components/script/dom/bluetoothuuid.rs:244]: Same

**Comment by dati91** [comment]
lgtm
r? @mmatyas

**Comment by mmatyas** [comment]
r+

**Pull request merged**